content/ai/ai.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: 'AI'
+Description: 'Artificial Intelligence (AI) refers to both the study of intelligent agents and to the intelligent agents themselves. Currently, AI can be categorized into three groups: narrow-, general-, and super-artificial intelligence.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/artificial-intelligence'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/machine-learning-ai-engineering-foundations'
+---
+
+**Artificial Intelligence (AI)** refers to both the study of intelligent agents and the intelligent agents themselves. An "intelligent agent" is any device designed to achieve some goal, receive information from its environment as input, and output a response that maximizes the success of achieving said goal.
+
+Currently, AI can be categorized into three groups: narrow-, general-, and super-artificial intelligence.
+
+## AI Categories
+
+- Artificial Narrow Intelligence (ANI) or Weak AI, are systems that are considered the least computationally potent AI. These systems include much of the contemporary machine learning and deep learning models with singular and narrowed functions such as Object Classification and Speech Recognition.
+
+- Artificial General Intelligence (AGI) or Strong AI, are systems that would pass the Turing Test, with intelligence that produces outputs that are indistinguishable from that of an adult human being. As of publication, no publicly known AGI has been developed.
+
+- Artificial Super Intelligence (ASI) or Superintelligence, is another form of AI yet to be developed that contains intelligence that can produce outputs that would vastly surpass the capacity of a human being.
+
+## History
+
+The first true instance of AI is arguable, with some determining the mechanism used to produce _Ars generalis ultima_ (The Ultimate General Art), published by [Ramon Llull](https://en.wikipedia.org/wiki/Ramon_Llull) in 1308 was an artificial intelligence with the mechanical means to create new knowledge from logic and complex mechanical techniques.
+
+In 1914, Spanish engineer [Leonardo Torres y Quevedo](https://en.wikipedia.org/wiki/Leonardo_Torres_y_Quevedo) demonstrates the first chess-playing machine in Paris, capable of receiving information about a chess game and playing a king and rook endgame against the king from any position without the aid of human intervention.
+
+In 1950, [Alan Turing](https://en.wikipedia.org/wiki/Alan_Turing) publishes _Computing Machinery and Intelligence_, introducing with it, the concept of the "imitation game". This would later be known as the Turing Test, which tests a machine's ability to display behavior and produce output that is indistinguishable from that of an adult person.
```

Comment:
```suggestion
In 1950, [Alan Turing](https://en.wikipedia.org/wiki/Alan_Turing) publishes "Computing Machinery and Intelligence", introducing with it, the concept of the imitation game. This would later be known as the Turing Test, which tests a machine's ability to display behavior and produce output that is indistinguishable from that of an adult person.
```